src/serenity/config.hpp

```diff
@@ -140,11 +121,11 @@ class SerenityConfig {
    * Sets CfgVariant config value.
```

Comment:
set (: For the first time it adds field to map, but next evaluations will modify the field so... `set` IMO